### PR TITLE
Bugfix/ios icons

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -39,24 +34,7 @@ func Run() {
...
-			logLevelStr = "info"
-		}
-
-		level, err := log.ParseLevel(logLevelStr)
-		if err != nil {
-			return err
-		}
-		log.SetLevel(level)
-
+		log.SetEnableDebugLog(true)
```
Comment (BirmacherAkos) [inline cli/cli.go]: Do we want to use debug log always?

Comment (lpusok) [reply cli/cli.go]: yes

File: cli/cli.go [hunk 2]
```diff
@@ -67,6 +45,7 @@ func Run() {
 	}
 
 	if err := app.Run(os.Args); err != nil {
-		log.Fatal(err)
+		log.TWarnf("%s", err)
```
Comment (godrei) [inline cli/cli.go]: why warn? afaik log.Fatal was red colored.

File: scanners/ios/icon.go [hunk 3]
```diff
@@ -23,16 +23,21 @@ func lookupIconBySchemeName(projectPath string, schemeName string, basepath stri
 		return nil, fmt.Errorf("scheme (%s) not found in project", schemeName)
 	}
 
-	mainTarget, err := mainTargetOfScheme(project, scheme.Name)
-
+	mainTarget, found, err := mainTargetOfScheme(project, scheme.Name)
+	if err != nil {
+		return nil, fmt.Errorf("main target not found for project (%s), scheme (%s), error: %s", project.Path, scheme.Name, err)
+	} else if !found {
+		log.TDebugf("No main target found for scheme: %s", scheme.Name)
```
Comment (godrei) [inline scanners/ios/icon.go]: why did not include the project? above you did

File: scanners/ios/icon.go [hunk 4]
```diff
@@ -85,11 +94,11 @@ func lookupIconByTarget(projectPath string, target xcodeproj.Target, basepath st
 	return icons, nil
 }
 
-func mainTargetOfScheme(proj xcodeproj.XcodeProj, scheme string) (xcodeproj.Target, error) {
+func mainTargetOfScheme(proj xcodeproj.XcodeProj, scheme string) (xcodeproj.Target, bool, error) {
 	projTargets := proj.Proj.Targets
 	sch, ok := proj.Scheme(scheme)
 	if !ok {
-		return xcodeproj.Target{}, fmt.Errorf("Failed to find scheme (%s) in project", scheme)
+		return xcodeproj.Target{}, false, fmt.Errorf("Failed to find scheme (%s) in project", scheme)
```
Comment (godrei) [inline scanners/ios/icon.go]: i think the only one possible error just codes that the function was not able to find the scheme, what's the point of returning both a bool and an error with this information

**Pull request merged**